test/image/compare_images

```diff
@@ -21,6 +21,22 @@ do
   format="${filename##*.}"
 
   case "$format" in
+    json)  
+      diff "$1/$filename" "$2/$filename" > "$3/$filename.diff"
```

Comment:
do you want to do a more JSON-aware kind of diff here, or does the JSON that comes out of Orca already have sorted keys ?

Reply:
The order seems to be deterministic so it's not strictly necessary. If that ever changes, I will add JSON-aware diffing. It would be a nice addition indeed.

Reply:
If it comes to that we could install and run https://www.npmjs.com/package/json-diff. I went with built-in `diff `for the time being.